src/dispatch.rs

```diff
@@ -534,18 +534,18 @@ pub(crate) fn gen_dispatch_str(cmd: DispatchType, dispatch: bool) -> crate::Resu
         SetCursor(theme, size) => format!("{theme} {}", *size),
         FocusUrgentOrLast => "focusurgentorlast".to_string(),
         FocusCurrentOrLast => "focuscurrentorlast".to_string(),
-        ToggleSplit => "togglesplit".to_string(),
-        SwapWithMaster(param) => format!("swapwithmaster{sep}{param}"),
-        FocusMaster(param) => format!("focusmaster{sep}{param}"),
-        AddMaster => "addmaster".to_string(),
-        RemoveMaster => "removemaster".to_string(),
-        OrientationLeft => "orientationleft".to_string(),
-        OrientationRight => "orientationright".to_string(),
-        OrientationTop => "orientationtop".to_string(),
-        OrientationBottom => "orientationbottom".to_string(),
-        OrientationCenter => "orientationcenter".to_string(),
-        OrientationNext => "orientationnext".to_string(),
-        OrientationPrev => "orientationprev".to_string(),
+        ToggleSplit => "layoutmsg togglesplit".to_string(),
+        SwapWithMaster(param) => format!("layoutmsg swapwithmaster{sep}{param}"),
+        FocusMaster(param) => format!("layoutmsg focusmaster{sep}{param}"),
```

Comment:
I just want to ask if you could make sure this works with the binding code found in config.rs:
https://github.com/hyprland-community/hyprland-rs/blob/a8ca325b9accfe03028d2526093a971b3f30731a/src/config.rs#L133
Since it adds a `sep` (`,`). Which when using regular IPC isn't added. This is weird, but it saves a lot of boilerplate.
https://github.com/hyprland-community/hyprland-rs/blob/a8ca325b9accfe03028d2526093a971b3f30731a/src/dispatch.rs#L468-L470

Reply:
I've tried the following with different all the available params:
```rust
Dispatch::call(DispatchType::SwapWithMaster(
           hyprland::dispatch::SwapWithMasterParam::Child
         )).unwrap();
```
and
```rust
Dispatch::call(DispatchType::FocusMaster(
          hyprland::dispatch::FocusMasterParam::Master
        )).unwrap();
```
Both seem to work fine, though I haven't figured difference between `child`, `master` and `auto` while running my tests from a terminal.

Reply:
Yeah, but I mean creating a keybind with hyprland-rs, as it uses the same dispatcher code, just replaces the formatting because IPC and binds have a different format.

Reply:
I'm sorry, I've tried to test it with bindings but was not able to get even the binding from the example working:
```rust
hyprland::bind!(SUPER, Key, "t" => ToggleFloating, None).unwrap();
```

It returns `Ok(())` but there is nothing new in `hyprctl binds` or an actual behavior.

Reply:
> I'm sorry, I've tried to test it with bindings but was not able to get even the binding from the example working:
> 
> ```rust
> hyprland::bind!(SUPER, Key, "t" => ToggleFloating, None).unwrap();
> ```
> 
> It returns `Ok(())` but there is nothing new in `hyprctl binds` or an actual behavior.

I'm sorry if this is too much to ask, but could you maybe add a println to check the command it's generating, and trying it with hyprctl since I'm not on hyprland at the moment. I really only want to merge this, if the binding works, as binding layout commands is pretty essential imo.